backend/src/schema/resolvers/user_management.js

```diff
@@ -5,7 +5,7 @@ import { neo4jgraphql } from 'neo4j-graphql-js'
 
 export default {
   Query: {
-    isLoggedIn: (parent, args, { driver, user }) => {
+    isLoggedIn: (_, args, { driver, user }) => {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-04T08:01:04Z" title="Thursday, July 4th 2019, 10:01:04 am +02:00">Jul 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -5,7 +5,7 @@ import { neo4jgraphql } from 'neo4j-graphql-js'
 
 export default {
   Query: {
-    isLoggedIn: (parent, args, { driver, user }) => {
+    isLoggedIn: (_, args, { driver, user }) => {
```


I would prefer `_parent` here, to keep clear where this parameter stands for. Just my opinion …

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-04T10:11:26Z" title="Thursday, July 4th 2019, 12:11:26 pm +02:00">Jul 4, 2019</time>_

---


I agree with you, I think it's not such a big deal cause there are other examples in the code base where someone looking for the naming would find it, but if we were to refactor all of them to have just an underscore, it would make it more difficult for people unfamiliar with the name convention to figure out what was going on